vm-systemd/misc-post-stop.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
-# Save default applications for DispVM
+# cleanup QubesIncoming/
+if [ -d /home/user/QubesIncomming ]; then
```

Comment:
```suggestion
if [ -d /home/user/QubesIncoming ]; then
```

And also don't hardcode `user`. See for example `qubes-rpc/qubes.WaitForSession` how to get the user name.